lib/ronin/web/browser/agent.rb

```diff
@@ -58,22 +58,22 @@ def initialize(visible:  false,
                   when Hash, nil then proxy
                   when URI::HTTP, Addressable::URI
                     {
-                      host:     proxy.host,
-                      port:     proxy.port,
-                      user:     proxy.user,
+                      host: proxy.host,
+                      port: proxy.port,
+                      user: proxy.user,
                       password: proxy.password
                     }
                   when String
                     uri = URI(proxy)
 
                     {
-                      host:     uri.host,
-                      port:     uri.port,
-                      user:     uri.user,
+                      host: uri.host,
+                      port: uri.port,
+                      user: uri.user,
                       password: uri.password
                     }
                   else
-                    raise(ArgumentError,"invalid proxy value (#{proxy.inspect}), must be either a Hash, URI::HTTP, String, or nil")
+                    raise(ArgumentError, "invalid proxy value (#{proxy.inspect}), must be either a Hash, URI::HTTP, String, or nil")
```

Comment:
I tend to omit the spaces in method calls. It saves time and doesn't impact readability that much (I think).